scripts/generate_vscode_config.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python3
+"""
+Generate VS Code MCP Server Configuration for Atlassian
```

Comment:
This script appears to duplicate the configuration generation found in `src/mcp_atlassian/utils/oauth_setup.py`. Please consider removing this file.